packages/fxa-auth-server/lib/senders/emails/templates/passwordChangeRequired/index.mjml

```diff
@@ -0,0 +1,36 @@
+<%# This Source Code Form is subject to the terms of the Mozilla Public
+  # License, v. 2.0. If a copy of the MPL was not distributed with this
+  # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
+
+<mj-include path="./css/global.css" type="css" css-inline="inline" />
```

Comment:
I know you know this now but we shouldn't need the global CSS included in templates, should be removed from all of these. 😎